aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -1207,6 +1208,7 @@ module aptos_framework::stake {
         voting_power_increase_limit: u64,
     ) {
         timestamp::set_time_has_started_for_testing(aptos_framework);
+        status::set_genesis_end_for_test(aptos_framework);
```

Comment:
Why we do we need to call this here?

Reply:
Yeah, we don't need it actually. I removed it.